### PR TITLE
[FR] Few markdown corrections

### PR DESCRIPTION
**Pull request by lbourdois**
I was optimistic that my previous PR would be the last. Again some corrections.

While checking the rendering on the site, I realized that for the 13-1 and 13-2 files, two English flag files were proposed.
I proposed a correction to fix that.

### CODE REVIEW

File: docs/fr/index.md [hunk 1]
```diff
@@ -328,11 +328,11 @@ D’après Yann Le Cun, ces cours sont destinés à des personnes de niveau bac+
 
 | Rôle | Photo | Contact | A propos |
 |:-----|:-----:|:--------|:------|
-|Instructeur|<img src="images/Yann.png" width="100" height="100">|<a href="https://twitter.com/ylecun">Yann Le Cun</a><br>yann@cs.nyu.edu|Silver Professor en Computer Science à l'Université de New York<br>et lauréat du prix Turing 2019
+|Instructeur|<img src="images/Yann.png" width="100" height="100">|<a href="https://twitter.com/ylecun">Yann Le Cun</a><br>yann@cs.nyu.edu|Silver Professor en Computer Science à l'Université de New York (NYU) <br> et lauréat du prix Turing 2019|
```
Comment (Atcold) [inline docs/fr/index.md:331]:
```suggestion
|Instructeur|<img src="images/Yann.png" width="100" height="100">|<a href="https://twitter.com/ylecun">Yann Le Cun</a><br>yann@cs.nyu.edu|Silver Professor en Computer Science à la New York University (NYU) <br> et lauréat du prix Turing 2019|
```

**Review by Atcold — CHANGES_REQUESTED** [review]
Small edit.

**Comment by lbourdois** [comment]
Done.

**Review by Atcold — APPROVED** [review]
Great, thanks!

**Pull request merged**